source/Playnite/GamesEditor.cs

```diff
@@ -328,6 +328,12 @@ public void ToggleFavoriteGame(List<Game> games)
             }
         }
 
+        public void SetCompletionStatus(Game game, CompletionStatus status)
+        {
+            game.CompletionStatus = status;

```

Comment:
Can you add check if new status is actually different and skip game update if it's the same?